bip-update-process.md

```diff
@@ -0,0 +1,516 @@
+License: CC0-1.0
+Replaces: 2
+```
+
+## tk: More input requested on following open questions
+
+- Should we have a Final and/or Active Status?
+- Should we add a mandatory Change Log section to BIPs?
+- Should we introduce versioning/revisions to (final) BIPs?
+
+## Abstract
+
+A _Bitcoin improvement proposal (BIP)_ provides a concise, self-contained, technical description of one new concept,
+feature, standard, implementation guideline, best-practice, or incident report relevant to the Bitcoin community.
+
+BIPs are intended to be the primary mechanism for proposing new protocol features, coordinating client standards, and
+documenting design decisions that have gone into Bitcoin implementations. Each BIP is primarily owned by its authors,
+the _BIP champions_[^champ], and represents the champions’ opinion or recommendation. The champions are expected to foster
+discussion, address feedback and dissenting opinions, and, if applicable, advance adoption of their proposal within the
+Bitcoin community.
+
+The BIPs repository serves as a highly visible platform to put forward mature proposals. The repository’s visibility
+facilitates the distributed consideration of proposals by establishing the current state of the proposal. The
+repository’s versioning of the text-based BIPs retains the historical record of each proposal.
+The BIPs repository does not[^acceptance] aim to track acceptance, adoption, or community consensus on BIPs except to facilitate a
+quick overview of various BIPs’s statuses to repository visitors.
+
+This BIP replaces BIP 2 with a more well-defined and clear process, and may be amended to address future needs of the
+BIP process.
+
+## What is a BIP?
+
+### What is the significance of BIPs?
+
+Individual BIPs do not represent Bitcoin community consensus or a general recommendation for implementation. A BIP
+represents a personal recommendations by the BIP champions to the Bitcoin community. Some BIPs may be adopted by one or
+multiple implementations or even incorporated into the Bitcoin protocol.
+
+### What should be documented in a BIP?
+
+The BIPs repository collects any proposals relevant to the Bitcoin community.
+
+OR:
+
+The BIPs repository is focused on information and technology that supports the bitcoin currency.[^astroturfing]
+
+## BIP format and structure
+
+### Specification
+
+BIPs should be written in mediawiki or markdown format.
+
+Each BIP should have the following parts:
+
+* Preamble -- Headers containing metadata about the BIP (see the section [BIP header preamble](#bip-header-preamble)
+  below).
+* Abstract -- A short (~200 word) description of the technical issue being addressed.
+* Motivation -- The motivation is critical for BIPs. It should clearly explain what issue the BIP addresses, and how the
+  existing situation is inadequate to address the problem that the BIP solves.
+* Rationale -- The rationale fleshes out the specification by describing what motivated the design and why particular
```

Comment:
Motivation covers the high-level rationale already, better for the detailed rationale to come after the Specification, I think?